packages/amplify_datastore/example/ios/Podfile

```diff
@@ -31,6 +31,11 @@ target 'Runner' do
   use_frameworks!
   use_modular_headers!
 
+  # Point pods to internal source files
```

Comment:
What is the purpose of these changes? Will customers have to make changes in their app?